doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -304,6 +304,7 @@ Restart CORTX
    </details>
    
 Tested by:
+- May 10, 2021: Shiji Zhang (shiji.zhang@tusimple.ai) using OVA release 1.0.4 on KVM 5.1
```

Comment:
can you correct the same on next line.

moreover, its good to see, you had tested OVA 1.0.4 on KVM 5.1 so can this be documented as well?